portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*Global variables pertaining to parallax animation*/
+var lFollowX = 0,
+    lFollowY = 0,
+    lFollowZ = 0,
+    x = 0,
+    y = 0,
+    friction = 1 / 10;
+
+$(document).ready(function() {
+  /* The animate function changes the picture perspective*/
+  function animate() {
+    
+    //Gets rotation values x and y
+    x += (lFollowX - x) * friction;
+    y += (lFollowY - y) * friction;
+    $('#profile').css({
+        'transform': 'perspective(600px) rotateY(' + -x + 'deg) rotateX(' + y + 'deg)'
+    });
+    window.requestAnimationFrame(animate);
+  }
+  
+  /* Detects the inciting parallax animation action */
+  $("#profile").on('mousemove click', function(e) {
+        /* Adjusting appropriate transform variables based on current mouse location*/
```

Comment:
`appropriate transform variables` isn't clear to me here. Can you be more specific? Also, clearer variable names may help address this.